hostthedocs/getconfig.py

```diff
@@ -48,7 +48,10 @@ def get(attr, default):
 
 
 def serve_gevent(app):
-    from gevent.pywsgi import WSGIServer
+    try:
+        from gevent.pywsgi import WSGIServer
+    except ModuleNotFoundError:
```

Comment:
Use an ImportError for compat please.

Reply:
I'm not sure why you would want to - if any other error import error besides "this module wasn't found" occurs, surely we'd rather that bubble up and be exposed to the user, because it isn't the known condition we're dealing with specifically here.

Reply:
ModuleNotFoundError was added in 3.6 and I'd rather not have a module on the import path that is only in newer versions (though I guess we have 3.6 in the pipfile, I don't think there's a reason that's the minimum version).
I'm open to more declarative solutions in the config but this would be good to have as well, I think.

Reply:
Oh!  I see, I guess I'm just spoiled by working in 3.8 these days, I didn't even realize.  I'll update it.